packages/shopify-codemod/transforms/global-reference-to-import.js

```diff
@@ -146,12 +155,17 @@ export default function globalReferenceToImport(
         });
 
       for (const anImport of imports.values()) {
-        const {file, name} = anImport;
+        const {file, name, hasNamedExports} = anImport;
         const {node: {body}} = path;
+        const importSpecifierBuilder = hasNamedExports ? (
```

Comment:
Something like:
```js
j.importDeclaration([
  (hasNamedExports ? j.importNamespaceSpecifier : j.importDefaultSpecifier)(j.identifier(name)),
],
```
?

That way I don't have to store it, but can use the returned result and immediately invoke it.